airflow/providers/salesforce/example_dags/example_tableau_refresh_workbook.py

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+This is an example dag that performs a refresh operation on a Tableau Workbook aka Extract. Since this is an
+asynchronous operation we don't know when the operation actually finishes. That's why we have a second task
+that checks exactly that. So that you can perform further operations after the extract has been refreshed.
+"""
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.providers.salesforce.operators.tableau_refresh_workbook import TableauRefreshWorkbookOperator
+from airflow.providers.salesforce.sensors.tableau_job_status import TableauJobStatusSensor
+from airflow.utils.dates import days_ago
+
+DEFAULT_ARGS = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': days_ago(2),
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False
+}
+
+with DAG(
+    dag_id='example_tableau_refresh_workbook',
+    default_args=DEFAULT_ARGS,
+    dagrun_timeout=timedelta(hours=2),
+    schedule_interval=None,
+    tags=['example'],
+) as dag:
+    task_refresh_workbook = TableauRefreshWorkbookOperator(
```

Comment:
I forgot to update this example. If we now have `block=True` by default means it will wait until the extract has been refreshed, we don't need the second task.

Or I add `block=False`

Reply:
I prefer the first one, but I'll leave it up to you :-)